setup.py

```diff
@@ -62,8 +62,8 @@
         packages=PACKAGES,
         include_package_data=True,
         entry_points={
-            'console_scripts': [
-                'recOrder.reconstruct = recOrder.cli_module:main'
-            ]
+            'console_scripts': ['recOrder.reconstruct = recOrder.cli_module:main',
+                                'recOrder-napari = scripts.launch_napari:main'],
+            'napari.plugin': 'recOrder-Calibration = recOrder.plugin.napari_plugin_entry_point'
```

Comment:
will probably need to rename this if the plugin will include more than just calibration